deploy/vagrant/Vagrantfile

```diff
@@ -39,17 +40,22 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         n.vm.provision :ansible do |ans|
           ans.playbook = "provision/playbook.yaml"
           ans.limit = "all" # to let "- hosts: all" work in playbooks
-          is_git = VersionYaml.type == "Github"
+          is_git = TachyonV.type == "Github"
```

Comment:
We should probably install git regardless of `is_git` now since spark requires git.